test/resetpassword.spec.js

```diff
@@ -0,0 +1,167 @@
+// Require the dependencies
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+import { User } from '../models';
+
+dotenv.config();
+
+
+// Configure chai
+chai.use(chaiHttp);
+const { expect } = chai;
+
+const validUser = {
```

Comment:
Consider extracting the test data to a `test\helpers\dummyData.js` file. Also, `verified: true` may not pass through the sign-up route again. Let's sync in person.